annex/annexA.md

```diff
@@ -5,15 +5,15 @@ This annex is the follow-up of examples related to chapter 1 in this tutorial.
 
 The example7.cpp is about: 
 - Listener to print the content of the event and the thread id
-- Publisher and subscriber for other types of data: int, my own struct such as Position(float _x, float _y)
+- Publisher and subscriber for other types of data: int, a customized struct such as Position(float _x, float _y)
 
 
-The example8.cpp is about: 
+The example8.cpp is an extension of example2.cpp from chapter1 of the tutorial. The goal of this new example is to publish a customized struct of data and then to apply some computations on them in order to publish another type of data. We have: 
 - 1 event loop
 - Publisher and subscriber for other type of data: in this case Position(float _x, float _y)
-- Distance to origin is computed and displayed by the subscriber
+- Distance to origin is computed and displayed by the subscriber based on the position received
 
-The example9.cpp is about:
-- Publisher and subscriber for other type of data: in this case Temperature (int t) and Position(float _x, _y)
+The example9.cpp is to test the event loop multithread behavior. Indeed, we have 2 threads, one that publishses and one that consumes the data. The goal of this example is to verify the multithreading capability of the event loop process. We have:
```

Comment:
publishses -->publishes. 

The goal is also to show that all the listeners of the event loop run in a single thread irrespective of the threading behaviour of the publishers.

Reply:
this goal is now added as well.